src/components/ErrorModal/ErrorModal.test.js

```diff
@@ -0,0 +1,218 @@
+import React from 'react';
+import {
+  fireEvent,
+  render,
+  screen,
+  cleanup
+} from '@testing-library/react';
+
+import '../../../test/jest/__mock__';
+
+import { FormattedMessage } from 'react-intl';
+
+import buildStripes from '../../../test/jest/__mock__/stripes.mock';
+import componentPropsCheck from '../../../test/jest/helpers/utils';
+
+import {
+  BACKEND_ERROR_CODES,
+  ERROR_MESSAGE_TRANSLATION_ID_BY_BACKEND_ERROR_CODE,
+} from '../../constants';
+
+import ErrorModal from '.';
+
+describe('ErrorModal', () => {
+  const onClose = jest.fn();
+  const openOverrideModal = jest.fn();
+  const testIds = {
+    errorModal: 'errorModal',
+    errorItem: 'errorItem',
+    overrideButton: 'overrideButton',
+    closeButton: 'closeButton',
+    messageToDisplay: 'messageToDisplay',
+  };
+  const baseError = {
+    message: 'Error message',
+    parameters: [{
+      value: 'testParameterValue',
+    }],
+  };
+  const baseProps = {
+    onClose,
+    onConfirm: () => null,
+    open: true,
+    errors: [],
+    stripes: buildStripes(),
+  };
+
+  const renderComponent = (props) => {
+    const combined = { ...baseProps, ...props };
+
+    render(<ErrorModal {...combined} />);
+  };
+
+  afterEach(cleanup);
+
+  it('should render error modal', () => {
+    renderComponent();
+
+    expect(screen.getByTestId(testIds.errorModal)).toBeInTheDocument();
+  });
+
+  it('should call onClose when clicking the close button', () => {
+    renderComponent();
+
+    fireEvent.click(screen.getByTestId(testIds.closeButton));
+    expect(onClose).toHaveBeenCalled();
+  });
+
+  describe('override button', () => {
+    it('should render if user has perms & errors fit', () => {
+      renderComponent({
+        errors: [{
+          ...baseError,
+          code: BACKEND_ERROR_CODES.itemNotLoanable,
+        }],
+        stripes: {
+          ...buildStripes(),
+          hasPerm: (arg) => arg === 'ui-users.overrideItemBlock',
```

Comment:
I would create constant for `'ui-users.overrideItemBlock'`. I see that you use it 3 times